addon/templates/components/frost-navigation.hbs

```diff
@@ -17,4 +16,7 @@
   <!-- User Menu -->
   {{component menu}}
 </div>
-{{liquid-modal}}
+{{to-elsewhere
```

Comment:
This would be similar to the binding here https://github.com/ciena-frost/ember-frost-modal/blob/master/addon/templates/components/frost-modal-binding.hbs